pkg/client/client.go

```diff
@@ -34,6 +35,7 @@ var (
 
 type Client struct {
 	ClientSet *kubernetes.Clientset
+	Pod       *v1.Pod
```

Comment:
can we find a way to avoid saving the `Pod` here?

Reply:
will calling `--cleanup` still work?

Reply:
Having the pod in the client wasn't efficient from the get-go. We already have the podname set in common which is the only place it's used in the exit code fetching. I've updated to remove the need for the entire pod. 

I removed fetching the pod exit code from when the cleanup flag is set. The program will exit with code 0.